src/models/IExtension.ts

```diff
@@ -1,25 +1,26 @@
 import { ILanguage } from './ILanguage';
 
-interface IExtension {
-  icon: string;
-  extensions: string[];
-  format: FileFormat;
-  alternate?: boolean;
-  light?: boolean;
+export interface IExtension {
```

Comment:
Common practice is to have every public `class`, `interface` on separate files.

We can then add an `index.ts` file inside the folder and `export` all `interfaces` from inside this file, gaining the advantage to reference only the folder path.